core/src/main/java/org/elasticsearch/cluster/routing/allocation/allocator/BalancedShardsAllocator.java

```diff
@@ -79,6 +81,9 @@
         Setting.floatSetting("cluster.routing.allocation.balance.threshold", 1.0f, 0.0f,
             Property.Dynamic, Property.NodeScope);
 
+    public static final Setting<Float> INDEX_SHARD_WEIGHT_MULTIPLIER_SETTING =
```

Comment:
I'm OK with floats here, eg the jump from 1 to 2 is quite big, I can imagine people wanting to use `1.5` here